legacy/cli/audit.go

```diff
@@ -47,6 +48,7 @@ func RunAuditCmd(opts *AuditOptions) error {
 		opts.RepoBranch,
 		opts.ManifestPath,
 		opts.UUID,
+		opts.Verbose,
```

Comment:
Do we usually backport changes to legacy?

Alos, this doesn't look like it's used anywhere?

Reply:
This is required to fill ServerContext's VerboseLogging field. Adding this field allows debugging statements to only log when the user provides the `--verbose` flag.

In short, it allows this function to exist:
```go
func (s *ServerContext) Debug(args ...interface{}) {
	if s.VerboseLogging {
		logrus.Debug(args...)
	}
}
```

Reply:
ACK, I wasn't sure if legacy was still in use 

/lgtm